### DISCUSSION TITLE
NordVPN - Allocating IP Manually

### DISCUSSION THREAD

**Discussion by WINOFFRG** [qdm12/gluetun#1189] [Q&A]
Hello! I am using NordVPN as VPN Provider because a site blocks my IP. When I run Nord VPN on Desktop and connect to `ca1190.nordvpn.com` Hostname and run the same site, It works perfectly. However, When I do the same thing on server a different IP Range is allocated for the same Hostname `ca1190.nordvpn.com` which is completly different and blocks my request with Error code 475.

Any idea how can I fix it?

**Comment by bnhf** [comment]
@WINOFFRG 

Might need a bit more detail on your setup, but my guess is there's something in your NordVPN Desktop configuration that you're not duplicating in Gluetun.  The VPN server would be found at the address (hard coded into Gluetun) for ca1190.nordvpn.com:
```
{
        "vpn": "openvpn",
        "region": "Canada",
        "number": 1190,
        "hostname": "ca1190.nordvpn.com",
        "tcp": true,
        "udp": true,
        "ips": [
          "139.28.218.211"
        ]
      },
```
Not sure how NordVPN allocates its exit nodes (which would be your public IP address), but it stands to reason that configuration differences could be a factor.

When I click on my gluetun container in Portainer, I can see all of the environment variables and how they are set:

![screenshot-nuc10-pc_9000-2022 10 10-15_32_54](https://user-images.githubusercontent.com/41088895/194956250-a268b2cb-d9f9-46e2-bb48-823ba8008b84.png)

I'd suggest comparing things like OPENVPN_CIPHERS, OPENVPN_PROTOCOL and OPENVPN_VERSION (in your list of environment variables, not my example above) as a starting point.

Perhaps you could post logs showing the OpenVPN connections as they're established for both methods you're using?

Also what website (or type of website) are you getting error code 475 from?
